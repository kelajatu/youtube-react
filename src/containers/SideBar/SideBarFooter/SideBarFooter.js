import React from "react";
import "./SideBarFooter.scss";

export function SideBarFooter() {
  return (
    <React.Fragment>
      <div className="footer-block">
        <div>About Press Copyright</div>
        <div>Creators Advertise</div>
        <div>Developers +MyTube</div>
        <div>Legal</div>
      </div>
      <div className="footer-block">
        <div>Terms Privacy</div>
        <div>Policy & Safety</div>
        <div>Test new features</div>
      </div>
      <div className="footer-block">
        <div>All prices include Tax</div>
      </div>
      <div className="footer-block">
        <div>Â© Jurgen Kela - A Youtube clone for portfolio purposes.</div>
      </div>
    </React.Fragment>
  );
}
