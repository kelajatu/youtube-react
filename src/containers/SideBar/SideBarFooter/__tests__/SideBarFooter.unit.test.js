import React from "react";
import { SideBarFooter } from "../SideBarFooter";
import { shallow } from "enzyme";

describe("SideBarFooter", () => {
  it("renders", () => {
    const wrapper = shallow(<SideBarFooter />);
    expect(wrapper).toMatchSnapshot();
  });
});
