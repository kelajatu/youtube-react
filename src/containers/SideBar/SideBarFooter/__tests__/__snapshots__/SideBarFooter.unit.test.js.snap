// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBarFooter renders 1`] = `
<Fragment>
  <div
    className="footer-block"
  >
    <div>
      About Press Copyright
    </div>
    <div>
      Creators Advertise
    </div>
    <div>
      Developers +MyTube
    </div>
    <div>
      Legal
    </div>
  </div>
  <div
    className="footer-block"
  >
    <div>
      Terms Privacy
    </div>
    <div>
      Policy & Safety
    </div>
    <div>
      Test new features
    </div>
  </div>
  <div
    className="footer-block"
  >
    <div>
      All prices include Tax
    </div>
  </div>
  <div
    className="footer-block"
  >
    <div>
      Â© Jurgen Kela - A Youtube clone for portfolio purposes.
    </div>
  </div>
</Fragment>
`;
